lib/http_server_fixture/mock_routes.ex

```diff
@@ -0,0 +1,8 @@
+defmodule HTTPServer.MockRoutes do
+  alias HTTPServer.Request
+  alias HTTPServerFixture.Handler
+
+  def route(%Request{path: "/post_test"} = req), do: Handler.SimplePost.handle(req)
+  def route(%Request{path: "/get_test"} = req), do: Handler.SimpleGet.handle(req)
+  def route(%Request{path: "/get_test_with_body"} = req), do: Handler.TestGetWithBody.handle(req)
+end
```

Comment:
these look like unit test mocks, they should probably be separated from the `http_server_fixture`(because that's for integration tests).  I suggest maybe getting a little more creative with folder structure to help organize that.  Typically all the unit test related mocks / supporting files all go in the `/test` folder for Elixir projects.

Reply:
for unit tests, you shouldn't have to actually start the server, where for the fixture you would. So the fixture code living in `/lib` makes sense to me.

Reply:
They were originally located in the test directory as a HTTPServerTest module, but it kept throwing an error that it couldn't access HTTPServerFixture.Handler when they were there. I'll keep playing around with it to see if there was something else that was preventing them from reaching it!

Reply:
[Unit tests sholdn't be using a HTTPServerFixture.Handler. I left a comment elsewhere that the Handler should be in the HTTPServer module.  The fixture Handlers should just implement it.](https://github.com/kelseyroy/http_server/pull/6#discussion_r1223381592) is related.